pkg/resolution/resolver/bundle/bundle.go

```diff
@@ -113,7 +113,7 @@ func GetEntry(ctx context.Context, keychain authn.Keychain, opts RequestOptions)
 			obj, err := readTarLayer(layerMap[l.Digest.String()])
 			if err != nil {
 				// This could still be a raw layer so try to read it as that instead.
-				obj, err = readRawLayer(layers[idx])
```

Comment:
maybe we forgot to return err here?

Reply:
I'm intentionally making no functional changes in this hygiene PR. I can hear an argument for changing the behavior, but that's beyond the scope of this PR. Happy to take care of that if you want to open an issue and assign it to me prior to approving.

Reply:
Oh I see, it totally makes sense!